code/modules/ruins/objects_and_mobs/sin_ruins.dm

```diff
@@ -110,7 +110,7 @@
 	name = "envy's knife"
 	desc = "Their success will be yours."
 	icon = 'icons/obj/wizard.dmi'
-	icon_state = "render"
+	icon_state = "envy"
 	item_state = "knife"
```

Comment:
inhand is still a kitchen knife 🤔

Reply:
how am i supposed to make inhands for this

Reply:
by getting good